### ISSUE TITLE
Updating `DialogHost.Avalonia` to v0.4 cause 'Could not load file or assembly exception'

### ISSUE THREAD

**Issue by dojo90** [AvaloniaCommunity/Material.Avalonia#163]
When updating the `DialogHost.Avalonia` to v0.4 I get a `TargetInvocationException`

```txt
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IO.FileLoadException: Could not load file or assembly 'DialogHost.Avalonia, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. The located assembly's manifest definition does not match the assembly reference. (0x80131040)
File name: 'DialogHost.Avalonia, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'
   at CompiledAvaloniaXaml.!AvaloniaResources.Populate:/DialogHost.axaml(IServiceProvider , Styles )
   at CompiledAvaloniaXaml.!AvaloniaResources.Build:/DialogHost.axaml(IServiceProvider )
   at CompiledAvaloniaXaml.!XamlLoader.TryLoad(String )
   --- End of inner exception stack trace ---
```

using Material.Avalonia **v2.5.0.73-nightly**

**Comment by SKProCH** [comment]
Hello, @dojo90 

This is fixed in version `3.0.0-rc0.85-nightly`. 
I will leave the issue open until the next release.

**Comment by SKProCH** [comment]
Fixed in `3.0.0-avalonia10-preview2`
